nomad/local/nomad-agent-conf.nomad

```diff
@@ -15,6 +15,11 @@ plugin "docker" {
       # Required for the bind mount for docker.sock
       enabled = true
     }
+
+    # We need net_admin for dnsmasq to work. Everything else should be default
```

Comment:
Out of curiosity, do you have a link to documentation or code that says what the default capabilities are?